SPECS/gpumon.spec

```
@@ -53,6 +53,9 @@ DESTDIR=%{buildroot} %{__make} install
 %{_unitdir}/xcp-rrdd-gpumon.service
 
 %changelog
+* Fri Nov 03 2023 Guillaume Thouvenin <guillaume.thouvenin@vates.tech> - 0.18.0-11.2
+- Don't redirect logs in daemon.log because there are already logged by RRDD
```

I started with minor fixes and ended-up with more changes to wording:

```suggestion
- Don't output to stdout/stderr (and thus daemon.log): they are already logged by RRDD
```